### DISCUSSION TITLE
파이썬 SolvedAC API Wrapper을 만들고 있습니다

### DISCUSSION THREAD

**Discussion by dev-ruby** [solvedac/unofficial-documentation#56] [일반]
```py
import SolvedAC
import asyncio

loop = asyncio.get_event_loop()

client = SolvedAC.SolvedClient()

profile = loop.run_until_complete(client.get_profile("devruby"))

print(profile.tier)
```

Output
```
Tier.Gold1
```

이런식으로 편리하게 사용이 가능한 API Wrapper을 만들고 있는데, 혹시 레포지토리에 추가하는건 어떨까요? Pypi에 올리려 합니다.

**Comment by RanolP** [comment]
깃허브 조직 초대 드렸습니다! 제가 이걸 관리할 시간이 안 나서 부족한 프로젝트인데도 관심 가져주셔서 감사합니다
solvedac/python 저장소에 접근 가능하게 만들어드렸으니 관련 작업물을 이쪽에 올려주시면 될 것 같습니다.

그와 별개로 제가 프로젝트와 관련해서 몇 가지 의견들이 있는데 뭔가 명문화를 한 건 아니라서 대략적으로 적어보자면

- 패키지명은 solvedac-community 혹은 solvedac_community일 것
- 가능하면 http client를 교체할 수 있을 것
- 패키지 관리는 poetry를 사용할 것 (파이썬)
- 타이핑을 최대한 정확히 할 것 (파이썬, 타입스크립트)
- 함수 이름으로 API 문서의 `operationId` 값과 최대한 매칭되는 이름을 설정할 것

이런 것들이 있습니다. 혹시 solved.ac 디스코드에 계시다면 개발 채널에서 @solvedac/unofficial-documentation 이란 이름으로 스레드를 파고 asdhugh1 (`@RanolP#4781`)을 멘션해주실 수 있을까요? 채팅방에서 논의를 진행해보면 좋을 것 같아요.
